.env.example

```diff
@@ -0,0 +1,6 @@
+DB_PORT=
```

Comment:
`DB_PORT` does not exist and is not used, even in #20
We are always using the default port for the mongo service, iirc

I see that you added the functionality to override, but I really don't see the point. If we use an external service that might be using another port, we would be using `MONGO_URI` anyway.
Sincerely, this just makes me want to scrap `DB_HOST` too and just use `MONGO_URI` for everything, but would have to look at the docker stuff, because I think this is like this due to some configs there.

Reply:
Agreed. That being said, i'm going with
```javascript
const MONGO_URI = process.env.MONGO_URI || "mongodb://localhost:27017";
```
removing DB_HOST and DB_PORT. `MONGO_URI` can be overwritten if needed.

Reply:
LGTM, this was probably a leftover from a rushed initial config, tbh, as other repos that use MongoDB only have the URI available as an env variable.

Just keep in mind that the docker config stuff might break, as it uses the env variables we are going to kill. However, some simple string interpolation should solve that, I hope. If you need help, feel free to ping me.